blueflood-http/pom.xml

```diff
@@ -72,6 +72,7 @@
         <artifactId>maven-surefire-plugin</artifactId>
         <version>2.14.1</version>
         <configuration>
+          <forkMode>always</forkMode>
```

Comment:
The static mocking using PowerMock doesn't seem to work without it, and anyway it makes each tests cleaner/more separated because every test suite is running in its own space.